app/views/site/_view_gene_expression_title_bar.html.erb

```diff
@@ -5,4 +5,14 @@
       <h3 class="help-block lead">Expression values are shown as the <strong id="selected-consensus"><%= params[:consensus].nil? ? 'mean' : params[:consensus] %></strong> of expression values across all genes for each cell.</h3>
     <% end %>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+<script>
+  <% if User.feature_flags_for_user(current_user).key?('ideogram_related_genes') && User.feature_flags_for_user(current_user)['ideogram_related_genes'] == true %>
```

Comment:
```suggestion
  <% if User.feature_flag_for_user(current_user, 'ideogram_related_genes') %>
```
If that doesn't work, let me know, because the intent of feature_flag_for_user is that it *should* work that way :)